.yarnrc

```diff
@@ -1 +1,2 @@
 registry "https://registry.yarnpkg.com"
+--install.ignore-optional true
```

Comment:
avoid installing fsevents

Reply:
just curious why this is needed. fsevents is used in chokidar for native macOS syscalls. this should help with perf when watching files

Reply:
I think the version of fsevents that's being installed is requiring python2